app/workers/fast_track/hypertension_observation_data.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionObservationData
+    attr_accessor :response
+
+    def initialize(response)
+      @response = response
+    end
+
+    def transform
+      entries = response.body['entry']
+      entries.map { |entry| transform_entry(entry) }
+    end
+
+    private
+
+    def pick(keys, hash)
+      hash.select { |k, _| keys.include? k }.with_indifferent_access
+    end
+
+    def transform_entry(raw_entry)
+      entry = pick(%w[issued component performer], raw_entry['resource'])
+      result = { issued: entry['issued'] }
```

Comment:
Could this and line 28 be combined? I don't see a reference to result anywhere else.